src/constants/sexOptions.js

```diff
@@ -2,18 +2,22 @@ export default [
   {
     value: 'male',
     labelId: 'MALE',
+    label: 'Male',
```

Comment:
I wanted to just use labelId, but that seems to involve a refactor of createFieldSchema itself, which seemed out of scope of this already-creeping ticket.